spec/features/shelters/show_spec.rb

```diff
@@ -117,5 +117,47 @@
       click_link "Edit Review"
       expect(current_path).to eq("/shelters/#{@shelter_1.id}/reviews/#{review_1.id}/edit")
     end
+
+    it 'I can see links to delete reviews next to each review' do
+      review_1 = @shelter_1.reviews.create!(
+        title: "Great Shelter!",
+        rating: 5,
+        content: "Has a lot of cats.",
+        image: "catshelter1.jpeg"
+      )
+      review_2 = @shelter_1.reviews.create!(
+        title: "Crappy Shelter!",
+        rating: 1,
+        content: "Not enough cats!",
+        image: "catshelter2.jpeg"
+      )
+      visit "/shelters/#{@shelter_1.id}"
+
+      expect(page).to have_link("Delete Review", count: 2)
+    end
+
+    it "When I click link to delete review the page reloads without review" do
+      review_1 = @shelter_1.reviews.create!(
+        title: "Great Shelter!",
+        rating: 5,
+        content: "Has a lot of cats.",
+        image: "catshelter1.jpeg"
+      )
+      review_2 = @shelter_1.reviews.create!(
+        title: "Crappy Shelter!",
+        rating: 1,
+        content: "Not enough cats!",
+        image: "catshelter2.jpeg"
+      )
+
+      visit "/shelters/#{@shelter_1.id}"
+
+      all(:link, 'Delete Review')[0].click
+
+      expect(current_path).to eq("/shelters/#{@shelter_1.id}")
+
+      expect(page).to_not have_content(review_1.attributes.values)
```

Comment:
This works for checking to make sure things aren't there but not for checking to see if they are because the values include id's.